app/api/monitoring.py

```
@@ -0,0 +1,31 @@
+        logger.warning(f"Unauthorized monitoring trigger attempt from IP. Token mismatch.")
+        raise HTTPException(status_code=403, detail="Forbidden: Invalid monitoring token")
+
+    try:
+        # Serverless(Cloud Run 등) 구동 환경에서는 응답 반환 후 CPU 할당이 제한되므로,
+        # BackgroundTasks를 사용하지 않고 동기적으로 로직을 끝까지 수행해야 완전한 실행이 보장됩니다.
+        # GCP Cloud Scheduler에서 실패를 즉각적으로 감지성(Visibility) 할 수 있도록 예외 시 500 에러를 반환합니다.
+        await send_discord_report()
+        return {"ok": True, "message": "Daily report triggered successfully"}
+    except Exception as e:
+        logger.error(f"Failed to trigger daily report: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail="Internal server error while triggering report")
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)